### ISSUE TITLE
gulp task not exporting globals

### ISSUE THREAD

**Issue by MikeFielden** [thlorenz/browserify-shim#37]
Ive been messing with this issue for some time and I just cant figure it out. Hopefully its not me :/

package.json
`
"gulp": "~3.5.2",
"gulp-browserify": "^0.5.0",
"debowerify": "^0.6.0",
"browserify-shim": "^3.3.2"
 "browserify": {
    "transform": ["debowerify", "browserify-shim" ]
  },
  "browserify-shim": {
    "./public/bower_components/jquery/dist/jquery.js": "$"
  },`

My `gulp` task looks like this:

`return gulp.src(INDEX_ENTRY_JS, {read: false})
  .pipe(browserify({
   debug: true,
   transform: ['debowerify','browserify-shim']
 }))
.pipe(size())
.pipe(require('gulp-rename')({suffix: "-bfy"}))
.pipe(gulp.dest(OUTPUT_SCRIPTS))
.pipe(livereload(server));`

Basically Im just hoping to get a simple `window.$` to be defined. I cant seem to make it work. 

Current error is:

`Error: browserify-shim needs to be passed a proper browserify instance as the first argument.
 you passed:'PATH TO THINGS'`

I've tried different versions and combinations of debowerify and this shim and none seem to work.
